examples/User Deprovisioning/Users_Deprovision.ps1

```diff
@@ -6,9 +6,10 @@
 ########################################################################################
 
 param (
-    [switch]$DryRun = $false # if enabled, then no delete/create/update calls will be made, only read ones
+    [switch]$DryRun = $false, # if enabled, then no delete/create/update calls will be made, only read ones
+    [string]$NewFilesOwnerID = "" # The ID of the user to transfer files to before deleting the user
 )
-
+Set-Alias box /Users/mcong/boxcli/bin/run;
```

Comment:
Looks like this part isn't needed for the script and this causes an error when you run script:

```
PS /Users/mgrytsai/IdeaProjects/boxcli/examples/User Deprovisioning> ./Users_Deprovision.ps1
box: /Users/mgrytsai/IdeaProjects/boxcli/examples/User Deprovisioning/Users_Deprovision.ps1:118
Line |
 118 |          return "$(box  configure:settings --help)" -like "*--analytic …
     |                    ~~~
     | The term '/Users/mcong/boxcli/bin/run' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the spelling of the name, or if a path was included,
     | verify that the path is correct and try again.

box: /Users/mgrytsai/IdeaProjects/boxcli/examples/User Deprovisioning/Users_Deprovision.ps1:118
Line |
 118 |          return "$(box  configure:settings --help)" -like "*--analytic …
     |                    ~~~
     | The term '/Users/mcong/boxcli/bin/run' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the spelling of the name, or if a path was included,
     | verify that the path is correct and try again.

PS /Users/mgrytsai/IdeaProjects/boxcli/examples/User Deprovisioning> ./Users_Deprovision.ps1
Importing csv of users to deprovision.
Successfully created new 'Employee Archive' root folder
```

Reply:
Ahhh, this one for debug only :'(